vector/src/main/java/im/vector/app/core/di/ScreenComponent.kt

```diff
@@ -153,6 +154,7 @@ interface ScreenComponent {
     fun inject(activity: ReAuthActivity)
     fun inject(activity: RoomDevToolActivity)
     fun inject(activity: TchapLoginActivity)
+    fun inject(activity: TchapHomeActivity)
```

Comment:
Don't you think we should remove here HomeActivity and LoginActivity?

Reply:
Yes but these activities implement inject method from ScreenComponent interface. So I can't

Reply:
TchapHomeActivity has been replaced by TchapHomeActivity